Extensions/Video/videoruntimeobject-pixi-renderer.js

```diff
@@ -149,13 +143,15 @@ gdjs.VideoRuntimeObjectPixiRenderer.prototype.pause = function() {
 
 /**
  * Set the loop on video in renderer
+ * @param {boolean} bool The new state of looped.
```

Comment:
@param {boolean} loop true to loop the video

Reply:
in vscode enable is white, is the argument i change it in the function ?